apps/bam.iobio/src/App.tsx

```diff
@@ -140,7 +139,12 @@ function App() {
 											key={key}
 											brokerKey={key}
 											title={BamDisplayNames[key]}
-											color={colors[index]}
+											styles={`
+												.iobio-histogram-title { text-align: left;} 
+												:host {
+													--iobio-data-color: ${colors[index]};
+												}
+												`}
```

Comment:
replying to your comment re string vs object... 
I'm not a fan of just a string. no DevExp help because it's just a string.
But you're right in that there's more work involved to use an object.